lib/commands/hasFocus.js

```diff
@@ -26,23 +26,34 @@
  *
  */
 
+import { RuntimeError } from '../utils/ErrorHandler'
+
 let hasFocus = function (selector) {
-    const finalSelector = selector === null ? this.lastPromise.inspect().value.selector : selector
-    let result = this.execute(function (selector) {
-        var focused = document.activeElement
-        if (!focused) {
-            return false
+    let result = this.elements(selector).then((res) => {
```

Comment:
So this command basically checks if one of the matching elements has focus (so it returns always a boolean value). Let's make sure to note that in the `@return` annotation